corehq/apps/sso/forms.py

```diff
@@ -583,9 +590,27 @@ class BaseSsoEnterpriseSettingsForm(forms.Form):
         label=gettext_lazy("Entity ID"),
         required=False,
     )
+    enable_user_deactivation = forms.BooleanField(required=False, label="Auto-Deactivation",
+                                help_text="This option ensures any authorization provided by CommCare HQ to"
+                                 " Web Users using SSO is removed when those users are removed from Azure AD"
+                                 " (Entra ID). This includes automatic deactivation of any API keys associated"
+                                 " with these users.",
+                                widget=BootstrapCheckboxInput(inline_label="Automatically deactivate Web Users"))
+    api_host = forms.CharField(required=False, label=gettext_lazy("Tenant Id"))
+    api_id = forms.CharField(required=False, label=gettext_lazy("Application ID"))
+    api_secret = forms.CharField(required=False, label=gettext_lazy("Client Secret"))
+    date_api_secret_expiration = forms.DateTimeField(required=False,
+                                                label=gettext_lazy("Secret Expires On"))
```

Comment:
This is probably design feedback, but I found it difficult to keep these variables straight in my head. When I was trying to figure out if the raw API Key was accessible to a tech-savvy user, I wasn't sure if I was supposed to be looking for `api_id` or `api_secret`. I think the intent was to keep these names generic so it could support other frameworks, so I just wanted to point out that others might get similarly confused as I did when looking at these generic names.

Reply:
Hi Matt, is the label helpful here? @biyeun Any thoughts regarding naming?

Reply:
@mjriley the end user does not see `api_secret` or `api_id`—these are all internal. As the spec outlined, we make sure that the user-facing text (in this case the `label`) matches the phrasing the user sees when copying that same data from Azure / Entra. The naming is carefully chosen to be identical to that experience. When we expand support for Okta and OneLogin, we will of course update the `label` based on what the user expects from those Identity Providers.